### PR TITLE
[KYIV-2023] add online-only devopsdays Ukraine

### PR DESCRIPTION
**Pull request by delgod**

### CODE REVIEW

**Comment by FloorD** [comment]
Screenshot (204)
The red banner still says "2022".

File: content/events/2023-kyiv/welcome.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
++++
+Title = "DevOpsDays #StandWithUkraine"
+Type = "welcome"
+aliases = ["/events/2023-kiev", "/events/2023-kyiv/", "/events/2023-ukraine/"]
+Description = "DevOpsDays #StandWithUkraine"
++++
+
+### Devopsdays [#StandWithUkraine](https://twitter.com/hashtag/StandWithUkraine)
+<div class="alert alert-danger" role="alert">
+   DevOpsDays Ukraine 2022 is now a charity edition of the conference with all proceeds going toward local, Ukrainian-led humanitarian causes.
```
Comment (mattstratton) [inline content/events/2023-kyiv/welcome.md:10]:
```suggestion
   DevOpsDays Ukraine 2023 is now a charity edition of the conference with all proceeds going toward local, Ukrainian-led humanitarian causes.
```

**Comment by delgod** [comment]
> Screenshot (204) The red banner still says "2022".

thanks, fixed

File: content/events/2023-kyiv/welcome.md (hunk 1 above)
Comment (delgod) [reply content/events/2023-kyiv/welcome.md]: thank you 🙇

**Pull request merged**